lib/client/order.rb

```diff
@@ -54,7 +75,14 @@ def get_shipments(order_id, params = {})
           response = client.get(uri({ order_id: order_id }, 'shipments'), params)
           return response.body
         rescue Faraday::Error => e
-          raise BigcommerceApiError.new('get order shipments', { order_id: order_id }, e)
+          raise BigcommerceApiError.new(
+            e.response[:status],
+            'get order shipments',
+            "Failed to get order shipments #{order_id}",
+            order_id,
+            { order_id: order_id },
```

Comment:
Drop the `order_id` in the data object, and add the BC errors.

Reply:
@mrogers-5s resolved